src/main/java/com/github/onsdigital/logging/layouts/PrettyJsonLayout.java

```diff
@@ -10,6 +12,10 @@
 
     @Override
     public String doLayout(ILoggingEvent event) {
-        return new PrettyJsonLogItem(event).asJson();
+        try {
+            return toPrettyJson(new JsonLogItem(event));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
Whats the reason to use a RuntimeException here? Moving towards using unchecked exceptions?

Reply:
I don't want the logging interface to force you to wrap the log call in a try catch - if logging fails we have bigger problems to worry about